spec/controllers/tasks_controller_spec.rb

```diff
@@ -252,6 +278,16 @@
         expect { put_update }.to change { task.reload.license.name }.to('new_license')
       end
 
+      it 'updates the tasks labels' do
+        expect { put_update }.to change { task.reload.labels }.from([existing_label]).to([new_label])
+      end
+
+      it 'creates new task label when requested' do
+        changed_attributes[:label_names] = ['some new label']
```

Comment:
this would fit in a new context, but you can probably just set it in the `let` in line 243

We may want to add an example again where we check if a new `Label` is created.

Reply:
I am not quite sure what you mean by that. Maybe you could make a code suggestion?

Reply:
see other comment